doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -451,3 +454,155 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
+       e.g. ahci,hd:/dev/sda.
+
+   * - "ahci-hd"
+     - It is an alias for "ahci"
+
+   * - "ahci-cd"
+     - Advanced Host Controller Interface used to connect with AT Attachment
+       Packet Interface device (for CD-ROM emulation). "ahci-cd" has the same parameters with "ahci".
+
+   * - "amd_hostbridge"
+     - Virtualized PCI AMD hostbridge
+
+   * - "hostbridge"
+     - Virtualized PCI hostbridge, a hardware bridge between the CPU's high-speed system local bus
+       and the Peripheral Component Interconnect (PCI) bus.
+
+   * - "pci_gvt"
+     - Intel GVT-g technology provides mediated device passthrough for Intel GPUs (Broadwell and newer).
+       It can be used to virtualize the GPU for multiple guest VMs, providing near-native graphics performance.
+
+   * - "virtio-blk"
+     - Virtio block type device, a string could be appended to specify the disk image,e.g.virtio-blk,/root/test.img,
+       user also could use "nodisk" to create a virtio-blk device with dummy backend.
+
+   * - "virtio-coreu"
+     - Used for Protected Audio Visual Path (PAVP) session management,with this support,
+       user VM is provided Protected Audio Visual Path service.
+
+   * - "virtio-input"
+     - Virtio type device to emulate input device, an virtio input decice evdev char device node
+       should be appended, e.g. -s n,virtio-input,/dev/input/eventX[,serial], “serial” is
+       an optional string. When it is specified, it will be used as the unique identification code
+       of guest virtio input device.
+
+   * - "virtio-ipu"
+     - Virtio ipu type device, like camera device.
+
+   * - "virtio-console"
+     - Virtio console type device for data input and output.
+
+   * - "virtio-hyper_dmabuf"
+     - Virtio device that allows sharing data buffers between VMs using a dmabuf-like interface
+
+   * - "virtio-hdcp"
+     - Virtio High-bandwidth Digital Content Protection (HDCP) type device. HDCP
+       is technology intended to protect unauthorized duplication of high
+       definition (HD) video and audio as it travels across connections.
+
+   * - "virtio-heci"
+     - Virtio Host Embedded Controller Interface, parameters should be appended with the format
+       <bus>:<device>:<function>,d<0~8>.
+
+   * - "virtio-i2c"
+     - Virtio i2c type device,parameters with format like
+       <bus>[:<client_addr>[@<node>]][:<client_addr>[@<node>]][,<bus>[:<client_addr>[@<node>]][:<client_addr>][@<node>]]
+       should be appended. <bus> specifies the bus number for the native I2C adapter, e.g.2 means /dev/i2c-2.
+       <client_addr> specifies the address for the native client devices such as 1C, 2F...
+       "@" specifies the prefix for the ACPI node.
+       <node> specifies the ACPI node name supported in the current code. Currently,only cam1,
+       cam2, and hdac are supported for APL platform. These nodes are platform-specific.
```

Comment:
Do we only support APL platforms for this parameter?

We don't actively support APL anymore.

What are these `cam1, `cam1` and `hdac` devices?

Reply:
cam1 cam2 hdac are APL platform APCI node in DSDT table